src/module/Entity/src/Entity/Entity/Entity.php

```diff
@@ -253,6 +253,11 @@ public function isUnrevised()
         return $this->countUnrevised() > 0;
     }
 
+    public function isDead()
+    {
+        return !$this->getHead();
```

Comment:
Oh I see, then I suggest creating a `Versioning\Filter\HasHeadCollectionFilter`, that uses `$revision->hasHead()` for filtering instead of creating one explicitly for entities, as `hasHead` is part of the `RevisionInterface` signature